src/commands/pczt/inspect.rs

```diff
@@ -0,0 +1,202 @@
+                    .map(|output| *output.value())
+                    .collect();
+                Ok::<_, pczt::roles::verifier::SaplingError<()>>(())
+            })
+            .expect("no error")
+            .with_orchard(|bundle| {
+                orchard_actions = bundle
+                    .actions()
+                    .iter()
+                    .map(|action| (*action.spend().value(), *action.output().value()))
+                    .collect();
+                Ok::<_, pczt::roles::verifier::OrchardError<()>>(())
+            })
+            .expect("no error")
+            .finish();
+
+        if !pczt.transparent().inputs().is_empty() {
+            println!("{} transparent inputs", pczt.transparent().inputs().len());
+            for (index, (hash_type, _, _, value)) in transparent_inputs.iter().enumerate() {
+                println!(
+                    "- {index}: {} zatoshis, {}",
+                    value.into_u64(),
+                    if hash_type == &SighashType::ALL {
+                        "SIGHASH_ALL"
+                    } else if hash_type == &SighashType::ALL_ANYONECANPAY {
+                        "SIGHASH_ALL_ANYONECANPAY"
+                    } else if hash_type == &SighashType::NONE {
+                        "SIGHASH_NONE"
+                    } else if hash_type == &SighashType::NONE_ANYONECANPAY {
+                        "SIGHASH_NONE_ANYONECANPAY"
+                    } else if hash_type == &SighashType::SINGLE {
+                        "SIGHASH_SINGLE"
+                    } else if hash_type == &SighashType::SINGLE_ANYONECANPAY {
+                        "SIGHASH_SINGLE_ANYONECANPAY"
+                    } else {
+                        unreachable!()
+                    },
+                );
+            }
+        }
+
+        if !pczt.transparent().outputs().is_empty() {
+            println!("{} transparent outputs", pczt.transparent().outputs().len());
+            for (index, value) in transparent_outputs.iter().enumerate() {
+                println!("- {index}: {} zatoshis", value.into_u64());
+            }
+        }
+
+        if !pczt.sapling().spends().is_empty() {
+            println!("{} Sapling spends", pczt.sapling().spends().len());
+            for (index, value) in sapling_spends.iter().enumerate() {
+                if let Some(value) = value {
+                    if value.inner() == 0 {
+                        println!("  - Dummy Spend");
+                    } else {
+                        println!("- {index}: {} zatoshis", value.inner());
+                    }
+                }
+            }
+        }
+
+        if !pczt.sapling().outputs().is_empty() {
+            println!("{} Sapling outputs", pczt.sapling().outputs().len());
+            for (index, value) in sapling_outputs.iter().enumerate() {
+                if let Some(value) = value {
+                    if value.inner() == 0 {
+                        println!("  - Dummy Output");
```

Comment:
We shouldn't be using zero values on the output side to identify things as "dummies"; zero-valued outputs are valid and they may have a memo attached. We should also make sure that this isn't being done in the firmware.